.gitignore

```diff
@@ -107,3 +107,6 @@ wp-tests-config.php
 
 # Visual regression test diffs
 tests/visual-regression/specs/__snapshots__
+
+# IDE generated folders
+.idea/
```

Comment:
@kofimokome You should have your local .gitginore file.
`git config --global core.excludesFile ~/.gitignore`
https://git-scm.com/docs/gitignore